cli/cmd/resources/profiles.go

```diff
@@ -75,17 +87,37 @@ var (
-		Dependencies:     []common.ProfileName{"db-payload-collection", "semconv", "category-attributes", "copy-scope", "hostname-as-podname", "java-native-instrumentations", "code-attributes", "query-operation-detector", "disableNameProcessorProfile", "small-batches"},
+		ShortDescription: "Bundle profile that includes db-payload-collection, semconv, category-attributes, copy-scope, hostname-as-podname, java-native-instrumentations, code-attributes, query-operation-detector, disableNameProcessorProfile, small-batches, size_m",
+		Dependencies:     []common.ProfileName{"db-payload-collection", "semconv", "category-attributes", "copy-scope", "hostname-as-podname", "java-native-instrumentations", "code-attributes", "query-operation-detector", "disableNameProcessorProfile", "small-batches", "size_m"},
+	}
+	ProfilesMap = map[common.ProfileName]Profile{
+		sizeSProfile.ProfileName:                      sizeSProfile,
+		sizeMProfile.ProfileName:                      sizeMProfile,
+		sizeLProfile.ProfileName:                      sizeLProfile,
+		fullPayloadCollectionProfile.ProfileName:      fullPayloadCollectionProfile,
+		dbPayloadCollectionProfile.ProfileName:        dbPayloadCollectionProfile,
+		queryOperationDetector.ProfileName:            queryOperationDetector,
+		semconvUpgraderProfile.ProfileName:            semconvUpgraderProfile,
+		categoryAttributesProfile.ProfileName:         categoryAttributesProfile,
+		copyScopeProfile.ProfileName:                  copyScopeProfile,
+		hostnameAsPodNameProfile.ProfileName:          hostnameAsPodNameProfile,
+		javaNativeInstrumentationsProfile.ProfileName: javaNativeInstrumentationsProfile,
+		codeAttributesProfile.ProfileName:             codeAttributesProfile,
+		disableNameProcessorProfile.ProfileName:       disableNameProcessorProfile,
+		smallBatchesProfile.ProfileName:               smallBatchesProfile,
+		kratosProfile.ProfileName:                     kratosProfile,
 	}
 )
 
 func GetAvailableCommunityProfiles() []Profile {
-	return []Profile{semconvUpgraderProfile, copyScopeProfile, disableNameProcessorProfile}
+	return []Profile{semconvUpgraderProfile, copyScopeProfile, disableNameProcessorProfile, sizeSProfile, sizeMProfile,
+		sizeLProfile}
 }
 
 func GetAvailableOnPremProfiles() []Profile {
-	return append([]Profile{fullPayloadCollectionProfile, dbPayloadCollectionProfile, categoryAttributesProfile, hostnameAsPodNameProfile, javaNativeInstrumentationsProfile, kratosProfile, queryOperationDetector, smallBatchesProfile},
+	return append([]Profile{fullPayloadCollectionProfile, dbPayloadCollectionProfile, categoryAttributesProfile,
+		hostnameAsPodNameProfile, javaNativeInstrumentationsProfile, kratosProfile, queryOperationDetector,
+		smallBatchesProfile, sizeSProfile, sizeMProfile, sizeLProfile},
```

Comment:
Community profiles are already added to enterprise

```suggestion
		smallBatchesProfile},
```